flotta-agent.spec

```diff
@@ -81,6 +83,9 @@ ln -sf %{_libexecdir}/yggdrasil/device-worker-race %{_libexecdir}/yggdrasil/devi
 systemctl enable --now nftables.service
 
 %changelog
+* Thu Jun 23 2022 Jordi Gil <jgil@redhat.com> 0.1.0-3
```

Comment:
if you wantt to update the release, do the same on the makefile and on the top of the file